### ISSUE TITLE
Swatches: expose Hex color code and swatch images

### ISSUE THREAD

**Issue by misha-kotov** [magento/graphql-ce#196]
### Description
As a Magento developer, I need GraphQL to expose the Hex color code and the swatch images for visual swatch attributes so that I can render the appropriate color/image on storefront. 

### Benefits
Easier rendering of visual swatches on storefront

### Additional information
Current request
```
{
  products(filter: {sku: {eq: "MH10"}}) {
    items {
      name
        ... on ConfigurableProduct{
        configurable_options{
          id
          label
          values {
            value_index
            label
            default_label
            store_label
            use_default_value
          }
        }
      }
    }
  }
}
```
Current response
```
{
  "data": {
    "products": {
      "items": [
        {
          "name": "Mach Street Sweatshirt ",
          "configurable_options": [
            {
              "id": 21,
              "label": "Color",
              "values": [
                {
                  "value_index": 49,
                  "label": "Black",
                  "default_label": "Black",
                  "store_label": "Black",
                  "use_default_value": true
                },
                {
                  "value_index": 50,
                  "label": "Blue",
                  "default_label": "Blue",
                  "store_label": "Blue",
                  "use_default_value": true
                },
...
```

Acceptance criteria
- GraphQL user is able to request the hex color code value of any swatch attribute that has a color selected
- GraphQL user is able to request the swatch image of any swatch attribute that has an image selected
- Docs are updated
- Tests are written

**Comment by VoronoyAlexandr** [comment]
Good day, can i try this task?

**Comment by naydav** [comment]
@VoronoyAlexandr
Ticket has been assigned to you

**Comment by magento-engcom-team** [comment]
Hi @misha-kotov. Thank you for your report. 
The issue has been fixed in magento/graphql-ce#443 by @sergiy-v in 2.3-develop branch
Related commit(s):
 - [9c4d6850c3d5f90d299c58c571d378e079feef29](https://github.com/magento/graphql-ce/commit/9c4d6850c3d5f90d299c58c571d378e079feef29)
 - [bdb506e88207b8c9ca892ebd0cd5915ae0b5edcb](https://github.com/magento/graphql-ce/commit/bdb506e88207b8c9ca892ebd0cd5915ae0b5edcb)
 - [cebac817f3f4cda03b0ded43adf6e1ad1523371f](https://github.com/magento/graphql-ce/commit/cebac817f3f4cda03b0ded43adf6e1ad1523371f)
 - [a1d671b2c688e734654cfaf26f00b22e05500612](https://github.com/magento/graphql-ce/commit/a1d671b2c688e734654cfaf26f00b22e05500612)
 - [4e2743b833db3d612d3be07b8aa5a1666baeaee2](https://github.com/magento/graphql-ce/commit/4e2743b833db3d612d3be07b8aa5a1666baeaee2)
 - [6dd69a5ba669c8af1649b689fe20bd8e496cd122](https://github.com/magento/graphql-ce/commit/6dd69a5ba669c8af1649b689fe20bd8e496cd122)
 - [fe7966978018bcb661486118d53e7c1e7889d5ec](https://github.com/magento/graphql-ce/commit/fe7966978018bcb661486118d53e7c1e7889d5ec)
 - [e6e0b71bca9036e005df4b3702fc3d8d7144e2ae](https://github.com/magento/graphql-ce/commit/e6e0b71bca9036e005df4b3702fc3d8d7144e2ae)
 - [06bbaaf38c1d53a4184c2be2b74de9225b33c39c](https://github.com/magento/graphql-ce/commit/06bbaaf38c1d53a4184c2be2b74de9225b33c39c)
 - [1e0e40116a2906e259bb32b814fa8fe03ada6cc1](https://github.com/magento/graphql-ce/commit/1e0e40116a2906e259bb32b814fa8fe03ada6cc1)
 - [931d0c56e1c6bea2baf91e3582ad3de8ddf51fa2](https://github.com/magento/graphql-ce/commit/931d0c56e1c6bea2baf91e3582ad3de8ddf51fa2)

The fix will be available with the upcoming 2.3.5 release.
